tests/functional/pvt/timings.test.js

```diff
@@ -320,39 +316,7 @@ function runPvtInStnTests (loader) {
 }
 
 function runClsTests (loader) {
-  testDriver.test(`LCP for ${loader} agent collects cls attribute`, supportsCLS, function (t, browser, router) {
```

Comment:
this seems like it should collect a CLS again (but with a positive attribute)

Reply:
This test would be remove anyways in the future?